src/renderer/components/ModalStepper/CloseButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Box, Button, Icons } from "@ledgerhq/react-ui";
+import styled from "styled-components";
+
+const CloseContainer = styled(Box).attrs(() => ({
+  top: 7,
+  right: 7,
+}))`
+  position: absolute;
```

Comment:
`position` seems also to be a prop of the `<Box />` component so it could be passed with `.attr` I think